src/schema/rules/files/raw/dwi.yaml

```diff
@@ -38,3 +38,22 @@ sbref:
     run: optional
     part: optional
     chunk: optional
+
+isotropic:
```

Comment:
Can you add a comment here summarizing what this group is?

Reply:
yeah looks good ! thx @Remi pointing to the rendered version :-)

Reply:
Scanner-derived images won't be isotropic if the DWI acquisition wasn't isotropic. Elsewhere in this file I see "parametric" vs. "nonparametric"; I wonder if that's the more suitable disambiguation here? Otherwise may need to adopt "scanner-derived" ([comment](https://github.com/bids-standard/bids-specification/pull/1831#issuecomment-2132483613) in #1831).